README.md

```diff
@@ -1,56 +1,119 @@
-# Getting Started with Cisco IOS-XE Smart licensing APIs
+# Getting Started with IOS-XE SLP REST APIs
 
-Sample code in Python that shows how to call IOS-XE SLP REST APIs. You can automate the usage reporting workflows using these APIs.
 
-For information about Smart Licensing Using Policy (SLP), see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).
+Sample Python script written in Python 3.8.5. The script demonstrates the use of IOS-XE SLP REST APIs to automate Smart Licensing Using Policy (SLP) workflows.
+
+SLP was first introduced on Cisco IOS-XE product instances in release Cisco IOS-XE Amsterdam 17.3.2. For more information, see [About Smart Licensing Using Policy](https://developer.cisco.com/docs/ios-xe/smart-licensing/#!introduction/about-smart-licensing-using-policy).
 
 ## Use Case Description
 
-For information about the purpose of each currently available API, see [About IOS-XE SLP REST APIs](ios-xe-smart-licensing-api/intro.md#about-ios-xe-slp-rest-apis).
+For information about each API that is demonstrated in this script, see [API Endpoint and Description](https://developer.cisco.com/docs/ios-xe/smart-licensing/#!about-the-apis/api-endpoint-and-description).
 
 Future enhancements include an API to return a Smart Licensing Authorization Code (SLAC).
 
 ## Installation
 
-Python must be installed. Further the following Python packages are required:
-* os
-* shutil
-* json
-* xmltodict
-* requests
-* yaml
-* tarfile
+Install [Python](https://www.python.org/downloads/) and the required packages. 
+
+* Python version: 3.8.5
+* Required Python packages:
+  * os
+  * shutil
+  * json
+  * xmltodict
+  * requests
+  * yaml
+  * tarfile
+  
 
 ## Configuration
-Configure the following where? we have to check CSSM API docs to know this and then add. 
 
-client_id: 55f65e59-df52-4073-bb12-4268d5167019 (the user has to enter their own;  will enquire how this should be done)
-client_secret: 78962244-9f2f-4e50-b233-f92adedd1e33 (the user has to enter their own; will enquire how this should be done)
-username: CSSM log in username (update config.yaml with dummy username)
```

Comment:
Remove the username and password.
I had a discussion with Nishanth earlier today, and from what I understood, the client_id and client_secret are sufficient.
No need to put the username and password in the script.

I also got the script to work for option (1), where it gets the list of SA/VA from CSSM.
I will upload the new version of the script in a couple of days.
This version does not use the username/password, and the config.yaml file also does not have the username/password.

Reply:
BTW, for this option to work, they have to set "cssm_connection: on" in config.yaml file.

Reply:
Thanks Sujay.  I've made the updates in this same draft -  please review. 
I think only this part still needs clarification (where exactly in apidocs-prod.cisco.com)
  client_id: <Your client_id obtained from apidocs-prod.cisco.com>
  client_secret: <Your client_secret obtained from apidocs-prod.cisco.com>

Reply:
The link to obtain client_id and client_secret is this:
https://apidocs-prod.cisco.com/?path=requestapiinfo
It has screenshots and step by step guide on how to obtain these two parameters.

Reply:
Thank you!.  Have added this to step 2. in the "configuration" section.